extensions/arc/package.json

```diff
@@ -989,11 +989,24 @@
                         "variableName": "AZDATA_NB_VAR_SQL_REPLICAS",
                         "options": {
                           "values": [
-                            "1",
-                            "3"
+                            "%arc.sql.two.replicas%",
```

Comment:
Do you really need to add "replica" to these? The label already says replicas so this seems a little redundant

Reply:
agreed, I think numbers are good enough here.